core/che-core-api-model/src/main/java/org/eclipse/che/api/core/model/workspace/config/ServerConfig.java

```diff
@@ -20,6 +21,12 @@
  */
 public interface ServerConfig {
 
+  /**
+   * {@link ServerConfig} and {@link Server} attribute name which can identify server as internal or
+   * external.
```

Comment:
I think it could be useful to specify the format of value for this attribute here.

Reply:
ok